services/web/server/tests/unit/with_dbs/10/meta/test_meta_core_repos.py

```diff
@@ -0,0 +1,115 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+from typing import Any, Callable, Dict
+from uuid import UUID
+
+import pytest
+from aiohttp import web
+from aiohttp.test_utils import TestClient, make_mocked_request
+from faker import Faker
+from simcore_service_webserver.constants import RQT_USERID_KEY
+from simcore_service_webserver.meta_core_repos import (
+    checkout_checkpoint,
+    create_checkpoint,
+    list_checkpoints,
+    update_checkpoint,
+)
+from simcore_service_webserver.meta_db import HEAD, VersionControlRepository
+from simcore_service_webserver.projects import projects_api
+
+ProjectDict = Dict[str, Any]
+
+
+# HELPERS
+
+
+# FIXTURES
+
+
+@pytest.fixture
+def aiohttp_mocked_request(client: TestClient, user_id: int) -> web.Request:
+    req = make_mocked_request("GET", "/", app=client.app)
+    req[RQT_USERID_KEY] = user_id
+    return req
+
+
+# TESTS
+
+
+@pytest.mark.acceptance_test
```

Comment:
what is the purpose with these acceptance_test(s)? I see that you mark these but what are you doing with them?

Reply:
it is a small experiment ... it is something i am reading in "robust python"